demos/sklearn/quickstart.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+
+from sklearn.base import BaseEstimator
+
+from coalescenceml.pipeline import pipeline
+from coalescenceml.step import step, Output
+from coalescenceml.integrations.constants import SKLEARN
+from coalescenceml.integrations.sklearn.step import (
+    SKLearnTrainConfig,
+    SKLearnTrainStep,
+)
+
+@step
+def importer() -> Output(
```

Comment:
Would we want to showcase the sklearn datasets? We can either be consistent and use the same dataset across integrations or maybe use integration datasets like sklearn's load_iris dataset or something.